dist/css/bootstrap.css

```diff
@@ -5264,7 +5264,7 @@ button.close {
   color: #fff;
   text-align: center;
   text-decoration: none;
-  background-color: rgba(0, 0, 0, .9);
+  background-color: #000;
```

Comment:
.9 means set alpha opacity to 90%

rgba is not supported by IE8 so I guess would be nicer to include both;

    background-color: #000;
    background-color: rgba(0, 0, 0, .9);